packages/editor/src/lib/editor/Editor.ts

```diff
@@ -712,9 +712,15 @@ export class Editor extends EventEmitter<TLEventMap> {
 		})
 
 		this.licenseManager = new LicenseManager()
-		const license = this.licenseManager.getLicenseFromKey(licenseKey)
 		this.watermarkManager = new WatermarkManager(this)
-		this.watermarkManager.checkWatermark(license)
+		this.licenseManager
```

Comment:
Need to use Promise syntax here. Subtle crypto uses async functions.

Reply:
let's use `await` syntax - makes it easier to read

Reply:
I would have to make the constructor async, is that possible/reccomended?

Reply:
ah yeah, let's not make the constructor async. but we can make a private async function:
`private async licenseChecks()` or something like that